documentation/homework/week3/todo/src/components/ListItem.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react'
+
+import Checkbox from './sub-components/Checkbox';
+import InputField from './sub-components/InputField'
+
+class ListItem extends Component {
+  // The Worst practice Ever
+  state = {
+    edit: false
+  }
+  
+  handleSubmit = (event) => {
+    const { id } = this.props
+    event.preventDefault()
+    this.props.handleUpdateListItem(event, id, 'description', event.target.description.value)
```

Comment:
You are not using controlled inputs! This is not the React way! You are accessing values as form data onSubmit instead of passing a `value` and a `onChange` prop to your input.

Additionally, because of this, when I click "edit" on a todo item, I see empty text fields instead of the existing values which should be there.

Please make these inputs controlled.